plex.json

```
@@ -1,59 +1,66 @@
-                        "index": 3
-                    }
-                }
-            }
-        },
-        "description": "Plex media server by Linuxserver.io",
-        "more_info": "<h4>Adding more media to Plex.</h4><p>You can add more Shares(with media) to Plex from the settings wizard of this Rock-on. Then, from Plex WebUI, you can update and re-index your library.</p>",
-        "ui": {
-            "slug": "web"
-        },
-        "volume_add_support": true,
-        "website": "https://hub.docker.com/r/linuxserver/plex/",
-        "version": "1.0"
-    }
-}
+{
+  "Plex-QuickSync": {
+    "containers": {
+      "plex-linuxserver.io-qs": {
+        "image": "linuxserver/plex",
+        "opts": [
+          [
+            "--net=host",
+            ""
+          ]
+        ],
+        "launch_order": 1,
+        "devices": {
+          "dev/dri": {
+            "description": "optional: Device Path for enabling Intel QuickSync (check CPU documentation at at ark.intel.com for availability) (/dev/dri). Leave blank if not needed.",
+            "label": "QuickSync device"
+          }
+        },
+        "ports": {
+          "32400": {
+            "description": "WebUI port. Suggested default: 32400",
+            "host_default": 32400,
+            "label": "WebUI port",
+            "ui": true
+          }
+        },
+        "volumes": {
+          "/config": {
+            "description": "Choose a Share for Plex configuration. E.g.: create a Share called plex-config for this purpose alone. It is important that the owner/group specified later have full access to the share (R/W/E)",
+            "label": "Config Storage"
+          },
+          "/data": {
+            "description": "Choose a Share for Plex content(your media). E.g.: create a Share called plex-data for this purpose alone. You can also assign other media Shares on the system after installation. This share needs to at least have Read/Execute access for the User group specified later",
+            "label": "Data Storage"
+          }
+   
+        },
+        "environment": {
+          "VERSION": {
+            "description": "Choose version of plex: Unless you know which version to try, just type latest",
+            "label": "VERSION",
+            "index": 1
+          },
+          "PUID": {
+            "description": "Enter a valid UID to run Plex with. It must have full permissions to all Shares mapped in the previous step.",
+            "label": "UID",
+            "index": 2
+          },
+          "PGID": {
+            "description": "Enter a valid GID to use along with the above UID. It (or the above UID) must have full permissions to all Shares mapped in the previous step.",
+            "label": "GID",
+            "index": 3
+          }
+        }
+      }
+    },
+    "description": "Plex media server by Linuxserver.io - with added (optional) Intel QuickSync hardware transcoding (check your processor's documentation).",

```

Harmonize style with other rock-ons and provide necessary information regarding docker image used, link to project website, and link to rock-on write-up in Rockstor's documentation:
```suggestion
    "description": "Organize and stream your personal media collection of movies, TV, music, and photos anywhere on all your devices. This rock-on supports hardware transcoding (optional) via Intel QuickSync -- see <a href='https://ark.intel.com/content/www/us/en/ark/search/featurefilter.html?productType=873&0_QuickSyncVideo=True' target='_blank'>Intel ark</a> to check your processor's compatibility.<p>Additional <a href='http://rockstor.com/docs/docker-based-rock-ons/plex-media-server.html#plex-server-rock-on' target='_blank'>documentation</a> is available for this rock-on.</p><p>Based on the docker image provided by Linuxserver.io: <a href='https://hub.docker.com/r/linuxserver/plex/' target='_blank'>https://hub.docker.com/r/linuxserver/plex/</a>.</p>",
```